spec/support/migration_name_helper.rb

```diff
@@ -0,0 +1,6 @@
+def require_migration
+  spec_name = Gem.location_of_caller.first
+  migration_name = spec_name.sub("spec/migrations", "db/migrate").sub("_spec.rb", ".rb")
+
+  require migration_name
+end
```

Comment:
If we made this return `File.basename(migration_name, '.rb').sub(/^[0-9]+_/, '').constantize`, and named it `migration_for_current_spec`, we could call it as:

```ruby
require "spec_helper"
describe migration_for_current_spec do
  # ..
```

Handy, or too magic?